webpack.embedding-sdk.config.js

```diff
@@ -18,6 +18,8 @@ const BUILD_PATH = __dirname + "/resources/embedding-sdk";
 const ENTERPRISE_SRC_PATH =
   __dirname + "/enterprise/frontend/src/metabase-enterprise";
 
+const skipDTS = process.env.SKIP_DTS === "true";
```

Comment:
Ah, so we always skip the ForkTsCheckerWebpackPlugin whenever we run `embedding-sdk:dev:webpack`

Reply:
Yeah, I designed the `embedding-sdk:dev:*` to be "sub-parts" of "embedding-sdk:dev", so the DTS files are already generated by the other command